distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1532,7 +1548,7 @@ type Table
     make_constant_column : Any -> Column
     make_constant_column self value =
         if Table_Helpers.is_column value then Error.throw (Illegal_Argument.Error "A constant value may only be created from a scalar, not a Column") else
-            Column.from_vector_repeated ("Constant_" + UUID.randomUUID.to_text) [value] self.row_count
+            Column.from_vector_repeated ("Constant_" + value.to_text) [value] self.row_count
```

Comment:
```suggestion
            Column.from_vector_repeated value.to_text [value] self.row_count
```

IMHO it is fine to not do the `Constant_` prefix.

Reply:
Done